pages/team-structure.js

```diff
@@ -0,0 +1,402 @@
+                    <NextSetButton onSelect={this.onDeterminePage}></NextSetButton>
+                </SystemComponent>)
+            } else {
+                return(
+                    <SystemComponent position="relative">
+                        <TeamHierMemberListGrid 
+                            tierId={tierId}
+                            roleCards={roleCards} 
+                            currentPage={this.state.page}
+                            onSelect={(cardId, tierId) => onSelect(cardId, tierId)}>
+                        </TeamHierMemberListGrid>
+                    </SystemComponent>
+                )
+            }
+        } else{
+            return(<React.Fragment></React.Fragment>)
+        }
+    }
+}
+
+
+class TeamHierarchy extends React.Component {
+
+    constructor(props){
+        super(props)
+        // NOTE: selectedMemberId -- each tier's card has a unique id
+        // selectedTier -- each tier has a unique ID
+        this.state = {
+            memberSelected: false,
+            selectedMemberId: undefined,
+            selectedTierId: 1,
+            selectedTeam: undefined,
+        };
+    }
+
+    onSelectMember = (cardId, tierId) => {
+        // TODO: put a condition ??
+        if (tierId === 2) {
+            const selectedSubTeamCard = subteam_json.filter(function(subTeamCard){return subTeamCard._id === cardId})[0]
+            this.setState({
+                selectedTierId: tierId, 
+                selectedMemberId: selectedSubTeamCard._id, 
+                memberSelected: true, 
+                selectedTeam: selectedSubTeamCard._id
+            })
+        } else {
+            this.setState({
+                selectedTierId: tierId, 
+                selectedMemberId: cardId, 
+                memberSelected: true, 
+                selectedTeam: undefined
+            })
+        }
+    }
+
+    render() {
+        return (
+            <PageTemplate title="Team Structure">
+                <React.Fragment>
+                <ThemeProvider theme={teamHierCustomTheme}>
+                    <SystemComponent 
+                        display="flex"
+                        overflow="hidden"
+                        height="100%"
+                        border="3px solid black"
+                    >
+                        <SystemComponent 
+                            width="240px"
+                            height="100%"
+                            bg="#D6D6D6"
+                        >
+                            <TeamStructSideNav></TeamStructSideNav>
+                        </SystemComponent>
+                        
+                        <TeamHierParentContainer pl={28} pr={28}>
+                            <TeamHierFormWrapper></TeamHierFormWrapper>
+                                
+                            <SystemComponent display="flex" flexDirection="column" overflowX="hidden">
+                                
+                                <HierSubSection>
```

Comment:
Hmm. The team structure is going to change often and it might change completely. Maybe we can consider a more dynamic approach? Maybe the team hierarchy should be dependent based on the database (For example, if we ever rename director to "Technical Director", then it should do that instead. So we could have sort of a numbering system for hierarchy (Director is 1, Lead is 2, Member is 3, etc.) and the UI could automatically render the right headers and hierarchy based on the numbering system.  This is just the first thing that came to my mind,  but there might be a better approach.